docs/UserGuide.md

```diff
@@ -32,103 +33,138 @@
 > **Command Format**
 > * Words in `UPPER_CASE` are the parameters.
 > * Items in `SQUARE_BRACKETS` are optional.
-> * Items with `...` after them can have multiple instances.
 > * The order of parameters is fixed.
 
 #### Viewing help : `help`
 Format: `help`
 
 > Help is also shown if you enter an incorrect command e.g. `abcd`
  
-#### Adding a person: `add`
-Adds a person to the address book<br>
-Format: `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...` 
+#### Adding an item: `add`
+Adds an item into the task manager<br>
+
+*With deadline*
+
+Format: `add NAME by DUE_DATE DUE_TIME [repeat every RECURRING_INTERVAL]`
+>Notice the `by` keyword? We use `by` to denote a deadline.
```

Comment:
how do you handle tasks names of the form: "meet zhiyuan by the coffee stall at canteen for a discussion on the project"